tests/LookupSqlTest.php

```diff
@@ -70,8 +70,8 @@ public function init(): void
             ->withTitle()
             ->addFields(['country_code' => 'code', 'is_eu']);
 
-        $this->hasMany('Friends', new LFriend())
-            ->addField('friend_names', ['field' => 'friend_name', 'concat' => ',']);
+//         $this->hasMany('Friends', new LFriend())
```

Comment:
concat no longer supported?

Reply:
No changes, simply tests on this are commented out so I commented it out. Reverted now.